src/crypto/CryptoInterface.c

```diff
@@ -89,18 +106,19 @@ int BSL_Crypto_ClearKeyHandle(void *keyhandle)
 {
     CHK_ARG_NONNULL(keyhandle);
 
-    BSLB_CryptoKey_t *key = (BSLB_CryptoKey_t *)keyhandle;
-    CHK_POSTCONDITION(BSL_SUCCESS == BSLB_CryptoKey_Deinit(key));
+    BSL_CryptoKey_t *key = (BSL_CryptoKey_t *)keyhandle;
+    CHK_POSTCONDITION(BSL_SUCCESS == BSL_CryptoKey_Deinit(key));
     BSL_FREE(key);
 
     return BSL_SUCCESS;
```

Comment:
Does this function also need to remove the key from the internal dict (if it is present)? The function docstring should be explicit about what this does or does not do with the internal dict.